src/Compilers/CSharp/Portable/Symbols/TupleTypeSymbol.cs

```diff
@@ -701,19 +847,26 @@ internal sealed class TupleFieldSymbol : FieldSymbol
         /// A tuple without backing fields is usable for binding purposes, since we know its name and type,
         /// but caller is supposed to report some kind of error at declaration.
         /// </summary>
-        public TupleFieldSymbol(string name, TupleTypeSymbol containingTuple, TypeSymbol type, FieldSymbol underlyingFieldOpt)
+        public TupleFieldSymbol(string name, TupleTypeSymbol containingTuple, TypeSymbol type, OneOrMany<FieldSymbol> underlyingFieldChainOpt)
```

Comment:
It is an overkill to keep and pass around the chain of fields. All we need to know in about the field is its final underlying field, the top level underlying tuple type for the owning TupleTypeSymbol and the field ordinal within the TupleTypeSymbol. With that information we can easy rebuild the chain during lowering.